client/me/purchases/confirm-cancel-domain/loading-placeholder.jsx

```diff
@@ -22,23 +22,22 @@ const ConfirmCancelPurchaseLoadingPlaceholder = ( { purchaseId, selectedSite } )
 
 	return (
 		<LoadingPlaceholder
-			title={ titles.confirmCancelPurchase }
+			title={ titles.confirmCancelDomain }
 			path={ path }>
-			<Card className="confirm-cancel-purchase-loading-placeholder__card">
-				<h2 className="loading-placeholder__content confirm-cancel-purchase-loading-placeholder__header" />
-				<div className="loading-placeholder__content confirm-cancel-purchase-loading-placeholder__subheader" />
-				<div className="loading-placeholder__content confirm-cancel-purchase-loading-placeholder__reason" />
-				<div className="loading-placeholder__content confirm-cancel-purchase-loading-placeholder__reason" />
+			<Card className="confirm-cancel-domain-loading-placeholder__card">
```

Comment:
These class names already didn't jive with our naming convention, but sub-components class names be named `.main-component__sub-component[-element]` per [this document](https://github.com/Automattic/wp-calypso/blob/00edbe3bb5b037ea4bd55933dbd72769637cd8ca/docs/components.md#example).

That would make this class: `confirm-cancel-domain__loading-placeholder-card`.